content/docs/ops/runbook/rotating-secrets.md

```diff
@@ -0,0 +1,44 @@
+---
+menu:
+  docs:
+    parent: runbook
+
+title: Rotating Secrets
+---
+
+# Start by rotating access to various services and IAM accounts.
+
+- Access AWS
+- Rotate these specific IAM Users first
+    - `bosh_user`
+    - `concourse_user`
+    - `cf_user`
```

Comment:
By the end of the quarter, we're supposed to get rid of all IAM users, so this shouldn't matter for long. But for now, 👍 to "rotate all the users" or similar.